kiwix-desktop.pro

```diff
@@ -82,6 +83,9 @@ isEmpty(PREFIX) {
 target.path = $$PREFIX/bin
 INSTALLS += target
 
+TRANSLATIONS = "resources/i18n/kiwix-desktop_fr.ts"
+CODECFORSRC = UTF-8
+
```

Comment:
It is just cosmetic, but maybe move them just after `FORMS`

Reply:
Ok